CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ Changelog for Critical Maps iOS
 - Fix: NavigationBar Colors under iOS 13 
 - Fix: UITableViewHeaderFooterView backgroundColor deprecation warning
 - Fix: Ambiguous auto layout constraints for Settings screen
+- Fix: Manage Friends tap to dismiss on name editing
```

Comment:
@mltbnz, @headione  Should we track the friends changes in the changeling? We just need to make sure that they don't end up in the store release notes yet.

Reply:
makes sense, I will remove it. I forgot it isn't in the master yet.

Reply:
👍. Sorry for the confusion and misleading issue template!